app/helpers/filter_routes_helper.rb

```diff
@@ -16,7 +16,16 @@ def statistical_data_sets_filter_path(*objects)
   end
 
   def filter_atom_feed_url
-    Whitehall::FeedUrlBuilder.new({ document_type: params.permit!.to_h[:controller].to_s }.merge(params.permit!.to_h)).url
+    Whitehall::FeedUrlBuilder.new(
+      params
+        .slice(:from_date, :page, :per_page, :to_date, :keywords, :locale, :include_world_location_news,
+               :official_document_status, :publication_type, :announcement_type, :format,
+               :publication_filter_option, :announcement_filter_option, :announcement_type_option,
+               taxons: [], subtaxons: [], topical_events: [], topics: [], departments: [], people: [], world_locations: [])
+        .permit!
+        .to_h
+        .merge(document_type: params[:controller].to_s),
```

Comment:
Similar point here about whether to_unsafe_hash is ok here to match previous functionality.

Reply:
Sure, unsafe_hash seems to work fine!